cabal-testsuite/PackageTests/ShowBuildInfo/A/cabal.project

```diff
@@ -0,0 +1 @@
+packages: .
```

Comment:
No newline at end of file :)

There's a bunch of other files in here that have this problem too. Mainly cosmetic I'll grant you but should still be fixed.